src/System Application/App/Edit in Excel/src/Filters/EditInExcelFilters.Codeunit.al

```diff
@@ -19,27 +19,60 @@ codeunit 1490 "Edit in Excel Filters"
     var
         EditinExcelFiltersImpl: Codeunit "Edit in Excel Filters Impl.";
 
+
+#if not CLEAN25
     /// <summary>
     /// Add the specified field using 'and' collection type.
     /// </summary>
     /// <param name="ODataFieldName">The OData name of the field referenced.</param>
     /// <param name="EditInExcelEdmType">The Edm type of the OData field as specified in the $metadata document.</param>
+    [Obsolete('Use AddFieldV2 instead.', '25.0')]
+#pragma warning disable AL0432
     procedure AddField(ODataFieldName: Text; EditInExcelEdmType: Enum "Edit in Excel Edm Type"): Interface "Edit in Excel Field Filter"
+#pragma warning restore AL0432
     begin
         exit(EditinExcelFiltersImpl.AddField(ODataFieldName, "Edit in Excel Filter Collection Type"::"and", EditInExcelEdmType));
     end;
+#endif
 
+    /// <summary>
+    /// Add the specified field using 'and' collection type.
+    /// </summary>
+    /// <param name="ODataFieldName">The OData name of the field referenced.</param>
+    /// <param name="EditInExcelEdmType">The Edm type of the OData field as specified in the $metadata document.</param>
+    procedure AddFieldV2(ODataFieldName: Text; EditInExcelEdmType: Enum "Edit in Excel Edm Type"): Interface "Edit in Excel Field Filter v2"
```

Comment:
Same thought as the other comment. To put the `interface` return as a var, making it simplier for developers in the future when it comes to obsoletion.